content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -881,24 +911,20 @@ quorum read to be served.
 | resourceVersionMatch unset            | limit unset                   | Most Recent           | Any                                       | Not older than                         |
-| resourceVersionMatch unset            | limit="n", continue unset     | Most Recent           | Any                                       | Exact                                  |
-| resourceVersionMatch unset            | limit="n", continue="<token>" | Continue Token, Exact | Invalid, treated as Continue Token, Exact | Invalid, HTTP `400 Bad Request`        |
+| resourceVersionMatch unset            | limit=\<n\>, continue unset     | Most Recent           | Any                                       | Exact                                  |
+| resourceVersionMatch unset            | limit=\<n\>, continue=\<token\> | Continue Token, Exact | Invalid, treated as Continue Token, Exact | Invalid, HTTP `400 Bad Request`        |
 | resourceVersionMatch=Exact[^1]        | limit unset                   | Invalid               | Invalid                                   | Exact                                  |
-| resourceVersionMatch=Exact[^1]        | limit="n", continue unset     | Invalid               | Invalid                                   | Exact                                  |
+| resourceVersionMatch=Exact[^1]        | limit=\<n\>, continue unset     | Invalid               | Invalid                                   | Exact                                  |
 | resourceVersionMatch=NotOlderThan[^1] | limit unset                   | Invalid               | Any                                       | Not older than                         |
-| resourceVersionMatch=NotOlderThan[^1] | limit="n", continue unset     | Invalid               | Any                                       | Not older than                         |
+| resourceVersionMatch=NotOlderThan[^1] | limit=\<n\>, continue unset     | Invalid               | Any                                       | Not older than                         |
 
 [^1]: If the server does not honor the `resourceVersionMatch` parameter, it is treated as if it is unset.
-| paging                          | resourceVersion unset | resourceVersion="0"                            | resourceVersion="{value other than 0}" |
-|---------------------------------|-----------------------|------------------------------------------------|----------------------------------------|
-| limit unset                     | Most Recent           | Any                                            | Not older than                         |
-| limit="n", continue unset       | Most Recent           | Any                                            | Exact                                  |
-| limit="n", continue="\<token\>" | Continue Token, Exact | Invalid, but treated as Continue Token, Exact  | Invalid, HTTP `400 Bad Request`        |
 
```

Comment:
To reviewers, line 892-897 was meant to be a nested table in the table footnote. However the table doesn't render.
This table is actually a copy of the first 3 rows from the table above. Since the footnote text "it is treated as if it is unset" is pretty clear, we don't need the nested table at all.

Reply:
Sounds good to me.

Reply:
:+1: 

As a general rule from a writing perspective: nested tables are a bad idea.

As a general rule from a markdown spec perspective: nothing but strings – no linebreaks, bulleted lists, or any other kind of formatting with line breaks – will render.

Reply:
Hi @tengqm .
I just looked at the generated tables. I don't see this string in the rendered output:
`If the server does not honor the `resourceVersionMatch` parameter, it is treated as if it is unset.`.
Perhaps, I am missed it.

Reply:
.... the one who missed it is me ...
seems I'm messing up a few branches and commits when switch back and forth .
pushing a new commit anyway.